arangopipe/arangopipe/build/lib/arangopipe/dataset_shift_detector.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
```

Comment:
Do you want to commit the build folder or should it be added to .gitignore?
This file is duplicated arangopipe/arangopipe/arangopipe/dataset_shift_detector.py